### PR TITLE
Chore add development setup to Readme.md

### PR DESCRIPTION
**Pull request by pawelkisielewicz**
Resolves https://github.com/Code-Poets/project-liberation/issues/35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+    ``` bash
+     pipenv install --dev
+    ```
+        
+    Alternatively, this can be also done using Pycharm IDE.  <br />
+    - Click "Add Interpreter" (right down corner) <br />
+    - Select Pipenv Environment <br />
+    - Choose python version, make sure that checkbox "Install packages from Pipfile" is selected <br />
+    - Confirm with OK <br />
+    New virtual environment will be created. Pycharm terminals will use this virtualenv, so there is no need to proceed step 3 any more. Skip to step 4
```
Comment (kbeker) [inline README.md]: Please delete this because this is unnecessary to simple development instruction.

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
 # project-liberation
+
```
Comment (kbeker) [inline README.md]: Please also add under `project-liberation` that this application is CodePoets home page.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+## Development setup
+
+### Preparing your environment for development
+
+1.  Install pipenv on your machine. Do not create any virtualenv, just install globally:
```
Comment (kbeker) [inline README.md]: ....just install **`pipenv`** globally on your operation system:

File: README.md [hunk 4]
```diff
@@ -1 +1,51 @@
 # project-liberation
+
+## Development setup
```
Comment (kbeker) [inline README.md]: Also please before `Development setup` please add some information about Python, Django and npm versions.

File: README.md [hunk 5]
```diff
@@ -1 +1,51 @@
...
+### Running tests and code analysis
+
+You can run automated tests and code analysis with:
+    ``` bash
+    ./full_check.sh
+    ```
+For details please see full-check.py file, it's description is always up-to-date 
+    
+## Production setup
+    TODO
```
Comment (kbeker) [inline README.md]: Please delete chapter `Production setup`. When we will need this we will create it :)

File: README.md (hunk 1 above)
Comment (pawelkisielewicz) [reply README.md]: I disagree, virtualenv can be created in two ways and IDE is useful tool for this. I will keep this, if you think it should be deleted anyway please remove it before merge.

File: README.md (hunk 5 above)
Comment (pawelkisielewicz) [reply README.md]: You mean... never? ;) I did it on purpose, if application is running in production development it should have Production setup. This way it is chance to do it. If I will remove it, this part will probably be never written.

File: README.md (hunk 1 above)
Comment (Jakub89) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (Jakub89) [reply README.md]: Done

**Pull request merged**